src/main/kotlin/no/nav/medlemskap/regler/v1/arbeidsforhold/HarForetaketMerEnn5AnsatteRegel.kt

```diff
@@ -19,13 +21,19 @@ class HarForetaketMerEnn5AnsatteRegel(
 
     override fun operasjon(): Resultat {
         if (arbeidsforhold.antallAnsatteHosArbeidsgivere(kontrollPeriodeForArbeidsforhold) finnesMindreEnn 6) {
-            statsborgerskap.registrerStatsborgerskapGrafana(kontrollPeriodeForArbeidsforhold, ytelse, regelId)
+            registrerDataForGrafana()
             return nei("Ikke alle arbeidsgivere har 6 ansatte eller flere")
         }
 
         return ja()
     }
 
+    fun registrerDataForGrafana(){
+        statsborgerskap.registrerStatsborgerskapGrafana(kontrollPeriodeForArbeidsforhold, ytelse, regelId)
```

Comment:
Hva er tanken bak å registrere statsborgerskap her? Registreres det også et annet sted? Og vil de to isåfall kunne gå i bena på hverandre/skape kluss i grafana?